oxidation/libparsec/crates/client_connection/src/config.rs

```diff
@@ -0,0 +1,21 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
+
+use libparsec_platform_http_proxy::ProxyConfig;
+
+#[derive(Default)]
+/// The parameters used to configure the http client.
+pub struct Config {
```

Comment:
Why do you need this ?

Reply:
I was thinking if we wanted to add more configuration to the http client, but right now only the proxy interest us